### DISCUSSION TITLE
Documentation on using UBL (no probe)

### DISCUSSION THREAD

**Discussion by wbvczar** [Jyers/Marlin#226] [Q&A] [answered]
I tried the pre-compiled 1.2.1 firmware (UBL-NoProbe) and I was excited to try UBL, but I was greeted with more menus and I couldn't figure out how to create my mesh.  When I homed the print head, it went to the center of the bed, versus the front left corner.

Is there any good instructions on how to use UBL on the Ender 3 v2?  The Marlin documentation didn't really help me.

Just looking for a good starting point to understand and use UBL.

**Comment by wbvczar** [comment, marked as answer]
After some more searching, I did find the following from the Marlin site on using UBL without a Z-probe.  I'll load the firmware up again and see if the menu options make sense with this new information.

https://marlinfw.org/docs/features/unified_bed_leveling.html#ubl-without-a-z-probe

**Reply by wbvczar** [reply]
No luck with using the Marlin documentation.  I just went back to the ManualMesh firmware, for now.  Maybe someone will do a write up on how to use the UBL with no Z-probe on the Ender 3 v2.

**Reply by Turismo05** [reply]
I would like this as well, can't seem to work it out.
